docs/posts/pydata-performance/dask_impl.py

```diff
@@ -0,0 +1,40 @@
+from __future__ import annotations
+
+import logging
+
+import dask.config
+import dask.dataframe as pd
+from dask.distributed import Client
+
+if __name__ == "__main__":
+    dask.config.set({"dataframe.convert-string": True})
+
+    with Client(silence_logs=logging.ERROR):
+        df = pd.read_parquet(
+            "/data/pypi-parquet/*.parquet",
```

Comment:
I'll make these paths a bit more friendly so that people run the workload from `$PWD`.